### PR TITLE
Specified that kong host names are case-sensitive.

### PR DESCRIPTION
**Pull request by sreeharisj23**
## Description

Specified that kong host names are case-sensitive.

## Related Issues

addresses #586

### CODE REVIEW

File: app/0.10.x/admin-api.md [hunk 1]
```diff
@@ -320,8 +320,9 @@ of `hosts`, `uris`, and `methods`. Kong will proxy all requests to the API to th
 ```
 HTTP 201 Created
 ```
-
+ 
 ```json
+When matching a request against an api's hosts field, the match is case sensitive
```
Comment (tlblessing) [inline app/0.10.x/admin-api.md:325]:
```suggestion
When matching a request against an api's hosts field, the match is case-sensitive.
```

File: app/0.10.x/getting-started/adding-your-api.md [hunk 2]
```diff
@@ -24,7 +24,8 @@ configuration of your Kong instance or cluster.
 
     Issue the following cURL request to add your first API ([Mockbin][mockbin])
     to Kong:
-
+    When matching a request against an api's hosts field, the match is case sensitive
```
Comment (tlblessing) [inline app/0.10.x/getting-started/adding-your-api.md:27]:
```suggestion
    When matching a request against an api's hosts field, the match is case-sensitive.
```

**Review by tlblessing — CHANGES_REQUESTED** [review]

**Comment by sreeharisj23** [comment]
I've made the changes you suggested @tlblessing

**Review by tlblessing — APPROVED** [review]
LGTM thank you Sreehari for the PR!

**Comment by falondarville** [comment]
This endpoint is deprecated. We are going to update the current documentation noting that `host` is case sensitive.

**Pull request closed without merging**